### DISCUSSION TITLE
momentum compaction factor error

### DISCUSSION THREAD

**Discussion by yysigari** [atcollab/at#555] [Q&A] [answered]
I am trying to run the touschekPiwinskiLifeTime.m file, and the section where it runs the mcf.m file gives this error:

> Error: File: ringpass.m Line: 104 Column: 1
> Illegal use of reserved keyword "end".
> Error in mcf (line 18)
> T = ringpass(RING,[Xdown Xup]);  

Apparently the ringpass.m doesn't exist anymore!
Any idea how to fix this?

**Comment by swhite2401** [comment, marked as answer]
Looking at the error message it seems you are try to overwrite a protected keyword "end", did you by any chance define a variable named "end"?

**Reply by yysigari** [reply]
Hey, thanks.

 I accidentally had typed something in front of one of the functions, problem solved!
